src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Food item priority level in the food inventory system.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPriority(String)}
+ */
+public class Priority {
+    public enum Level {
+        LOW,
+        MEDIUM,
+        HIGH
+    }
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Priority should be either high, medium or low.";
+    public final Level value;
+
+    /**
+     * Constructs a {@code Priority}.
+     *
+     * @param priority A valid priority level.
+     */
+    public Priority(String priority) {
+        priority = priority.toLowerCase();
```

Comment:
A little odd to change to lower case before validation checks. Maybe only do so in line 30 (i.e. switch(priority.toLowerCase())?

Reply:
This is to ensure that both upper and lower cases are accepted as a valid command. For example, "add d/tuna p/HIGH" or "add d/tuna p/high" or "add d/tuna p/High" are all valid commands.

Reply:
I understand, you are converting the input to lower case then performing validation checks on them. Then there is no need to change.